src/services/CapabilitiesManager.ts

```diff
@@ -41,14 +57,42 @@ export function hasTalkFeature(token: string = 'local', feature: string): boolea
  * @param key2 second-level key (e.g. 'allowed')
  */
 export function getTalkConfig(token: string = 'local', key1: keyof Config, key2: keyof Config[keyof Config]) {
+	const remoteCapabilities = getRemoteCapability(token)
 	if (localCapabilities?.spreed?.['config-local']?.[key1]?.includes(key2)) {
 		return localCapabilities?.spreed?.config?.[key1]?.[key2]
-	} else if (token === 'local' || !remoteCapabilities[token]) {
+	} else if (token === 'local' || !remoteCapabilities) {
 		return localCapabilities?.spreed?.config?.[key1]?.[key2]
 	} else {
 		// TODO discuss handling remote config (respect remote only / both / minimal)
-		return remoteCapabilities[token]?.spreed?.config?.[key1]?.[key2]
+		return remoteCapabilities?.spreed?.config?.[key1]?.[key2]
+	}
+}
+
+/**
+ * Returns capability for specified token (if already matches from one of remote servers)
+ * @param token token of the conversation
+ */
+function getRemoteCapability(token: string): RemoteCapability | null {
+	if (remoteCapabilities[remoteTokenMap[token]]) {
+		return remoteCapabilities[remoteTokenMap[token]]
+	}
```

Comment:
```
if (remoteTokenMap[token] && remoteCapabilities[remoteTokenMap[token]]) {
		return remoteCapabilities[remoteTokenMap[token]]
	}

```
It may not be generated yet.

Reply:
It should be generated, when the file is imported/called first time

Reply:
and what about creating the conversation? its token won't be there yet